tocgen.py

```diff
@@ -0,0 +1,20 @@
+from jinja2 import Environment, FileSystemLoader
+import os, re
+
+env = Environment(loader=FileSystemLoader(os.path.dirname('./')))
+template = env.get_template('toc.md.j2')
+
+rootDir = '.'
+
+list_dirs = ""
+for dirName, subdirList, fileList in os.walk(rootDir):
+    if re.match('^./.git',dirName):
```

Comment:
Since you don't want the `tests/` directory showing up in the table of contents either, you should adapt the above approach (listing directories in the top level of the repo) so it can also ignore directories with specific names (`.git` and `tests` being two).